docker-compose.yml

```diff
@@ -1,10 +1,18 @@
 version: "3.8"
 
 services:
+  client:
+    container_name: client_container
+    build:
+      context: ./client
+      dockerfile: Dockerfile
+    ports:
+      - "4200:4200"
+    command: ["npm", "run", "dev"]
   server:
     container_name: server_container
     build:
-      context: .
+      context: ./server
       dockerfile: Dockerfile
     environment:
       NODE_ENV: ${NODE_ENV}
```

Comment:
Can you validate that if we have `./server/.env` setup, it will add the env vars correctly please? A quick video should be good

Reply:
Not this week. Dont have time. Will send this PR out and wont be able to work on anything till Monday, feel free to do it yourself if that timeline is restricting.